apps/modernization-ui/src/apps/page-builder/pages/PageDetails/PageDetails.tsx

```diff
@@ -0,0 +1,78 @@
+import { Concept, Condition, PagesResponse } from 'apps/page-builder/generated';
+import { fetchConditions } from 'apps/page-builder/services/conditionAPI';
+import { fetchMMGOptions } from 'apps/page-builder/services/valueSetAPI';
+import { SelectInput } from 'components/FormInputs/SelectInput';
+import React, { useContext, useEffect, useState } from 'react';
+import { Button, Form, Icon } from '@trussworks/react-uswds';
+import { UserContext } from 'user';
+import './PageDetails.scss';
+import { PageDetailsField } from './PageDetailsField';
+import { fetchPageDetails } from '../../services/pagesAPI';
+import { useParams } from 'react-router-dom';
+
+export type FormValues = {
+    conditionIds: string[];
+    dataMartName?: string;
+    eventType: string;
+    messageMappingGuide: string;
+    name: string;
+    pageDescription?: string;
+};
+const eventType = [
+    { value: 'CON', name: 'Contact Record' },
+    { value: 'IXS', name: 'Interview' },
+    { value: 'INV', name: 'Investigation' },
+    { value: 'ISO', name: 'Lab Isolate Tracking' },
+    { value: 'LAB', name: 'Lab Report' },
+    { value: 'SUS', name: 'Lab Susceptibility' },
+    { value: 'VAC', name: 'Vaccination' }
+];
+
+export const PageDetails = () => {
+    const { state } = useContext(UserContext);
+    const { pageId } = useParams();
+    const [page, setPage] = useState<PagesResponse>({ name: '', id: 0, description: '' });
+    const [conditions, setConditions] = useState<Condition[]>([]);
+    const [mmgs, setMmgs] = useState<Concept[]>([]);
+    // const form = useForm<FormValues>({ defaultValues: { eventType: 'INV' }, mode: 'onBlur' });
+    const token = `Bearer ${state.getToken()}`;
+
+    useEffect(() => {
+        const token = `Bearer ${state.getToken()}`;
+        fetchMMGOptions(token)
+            .then((data) => {
+                setMmgs(data);
+            })
+            .catch((error: any) => {
+                console.log('Error', error);
```

Comment:
I don't think we should output this error to the console. We should remove the try/catch or we should track the error in state and create an error boundary. We could also use something like componentDidCatch to create that error boundary